<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.3">
    <title>Links for poetic-errors</title>
  </head>
  <body>
    <h1>Links for poetic-errors</h1>
<a href="https://files.pythonhosted.org/packages/4d/ae/e9688b928e7c3d5002775f914f2328ee51fccfda704c57d14974e9e9bf82/poetic_errors-0.1.2-py3-none-any.whl#sha256=cfc4f1fc7e41cdbf1198b4d7276f179f39b218841124fd598898bd615d8e2ada" data-requires-python="&gt;=3.6" data-dist-info-metadata="sha256=5ecae7ac67242040292e7467b88732ab4ce6cff19b9e5d26be205e3b9d98f59c" data-core-metadata="sha256=5ecae7ac67242040292e7467b88732ab4ce6cff19b9e5d26be205e3b9d98f59c">poetic_errors-0.1.2-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/22/fc/4cc32779b858ac5d58fe6c138e149b11436678c1653eaf7b1fd608b2785f/poetic_errors-0.1.2.tar.gz#sha256=df1149d8593bfc814eb0c9f330e80f26dc21da99c2a024f8e879a83ff925e919" data-requires-python="&gt;=3.6" >poetic_errors-0.1.2.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/87/58/b07500fb3863e4fc9285eceb1972a9eae03439034861ddf14a6bee7e94ea/poetic_errors-0.1.3-py3-none-any.whl#sha256=aac8fcd053edb000827d82d888ae796b4a77e0494b8c693e3f12bc2459e6ef17" data-requires-python="&gt;=3.6" data-dist-info-metadata="sha256=90262ccf69403f6dc395d8b3a2fe4d6c10cc2ca159426a99687d269f5b9c9002" data-core-metadata="sha256=90262ccf69403f6dc395d8b3a2fe4d6c10cc2ca159426a99687d269f5b9c9002">poetic_errors-0.1.3-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/4a/f9/2e2fc1881b6f2e8b5d127b3e5865389ad380a1b4bfd1b5dfcc889bcbf325/poetic_errors-0.1.3.tar.gz#sha256=024892fcb77a805b704e23d9eba48745059b538cba61b173b741cb47bd349032" data-requires-python="&gt;=3.6" >poetic_errors-0.1.3.tar.gz</a><br />
</body>
</html>
<!--SERIAL 26863422-->